system_test/common/aest_mempool_SUITE.erl

```diff
@@ -137,7 +137,7 @@ test_mempool_ttl_cleanup(Cfg) ->
 
     %% Check the first one got on the chain
     aest_nodes:wait_for_value({txs_on_chain, [DepositHash1]},
-                              [node1, node2], {blocks_delta, 2}, []),
+                              [node1, node2], {blocks_delta, 5}, []),
```

Comment:
Loosen up the expectation for the deposit to make it in the next 2 generations (with fast mining this can be a fragile assumption)